gadgets/trace_exec/program.bpf.c

```diff
@@ -35,6 +35,9 @@ struct event {
 	unsigned int args_size;
 	char cwd[MAX_STRING_SIZE];
 	char file[MAX_STRING_SIZE];
+	dev_t dev_major;
+	dev_t dev_minor;
```

Comment:
Both can be an `unsigned int`: it seems to be the underlying type suggested in https://github.com/torvalds/linux/blob/v6.12/include/linux/kdev_t.h#L7-L12